hexrd/imageseries/load/imagefiles.py

```diff
@@ -19,6 +19,8 @@
 from .metadata import yamlmeta
 from ..imageseriesiter import ImageSeriesIterator
 
+SCALE_FACTOR = 2.9452155399372724e-07
```

Comment:
Can we rename this to `GEL_SCALE_FACTOR` for clarity?

Reply:
Sure thing! Done